compiler/testData/integration/ant/jvm/fork/build.log.expected

```diff
@@ -0,0 +1,11 @@
+OUT:
+Buildfile: [TestData]/build.xml
+
+build:
+  [kotlinc] Compiling [[TestData]] => [[Temp]/fork.jar]
+  [kotlinc] -XX:CICompilerCount=4 -XX:InitialHeapSize=67108864 -XX:MaxHeapSize=134217728 -XX:MaxNewSize=33554432 -XX:NewSize=33554432 -XX:+PrintCommandLineFlags -XX:ReservedCodeCacheSize=67108864 -XX:+UseCompressedClassPointers -XX:+UseCompressedOops -XX:-UseLargePagesIndividualAllocation -XX:+UseParallelGC
```

Comment:
On our buildserver, both tests fail because we don't have the `-XX:-UseLargePagesIndividualAllocation` argument. Can we do something about it?

Actually these arguments can be quite different on different JDKs, and to be honest, I don't see how to make this test JDK-independent... Maybe we'd need some other "custom" kind of test, which won't check for the exact match of the output with the `.expected` file, but will look for some specific substring in the output to verify that the JVM args have been applied correctly (e.g. like just `-XX:CICompilerCount=4` in this case)?

Reply:
Or, as a dirty solution, we could change the output normalization logic in `AbstractAntTaskTest.normalizeOutput`, so that it would replace the string with a bunch of `-XX` arguments, printed by java, to a string containing only the known arguments?

So that this line:
```
-XX:CICompilerCount=4 -XX:InitialHeapSize=67108864 -XX:MaxHeapSize=134217728 -XX:MaxNewSize=33554432 -XX:NewSize=33554432 -XX:+PrintCommandLineFlags -XX:ReservedCodeCacheSize=67108864 -XX:+UseCompressedClassPointers -XX:+UseCompressedOops -XX:-UseLargePagesIndividualAllocation -XX:+UseParallelGC
```

would be transformed into something like:
```
-XX:CICompilerCount=4 [JavaXXArgs]
```

(similarly to how we replace test data path with `[TestData]`, time with `[time]`, etc.)

I don't like this though, since some JDKs can also print some -XX args before CICompilerCount.

Maybe there's some easier way to write this test (which is: to verify that JVM args have been applied) that I'm not seeing, I don't know.

Reply:
Thanks for you suggestions, it is indeed a tricky problem. 

According to [Oracle's documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/java.html) , it seems all ```Non-Standard Options```(start with -X) and ```Advanced Runtime Options```(start with -XX) are not guaranteed to be supported by all JVM implementations. 

So I reduced the number of parameters,  leaving only two parameters ```-Xms``` and ```-Xmx```, and hope common JVM implementations support them...

What do you think? is this feasible?

Reply:
Yeah, this looks like a great idea :). :+1: